browser/app/js/browser/__tests__/About.test.js

```diff
@@ -16,7 +16,7 @@
 
 import React from "react"
 import { shallow } from "enzyme"
-import { AboutModal } from "../AboutModal"
+import { About } from "../About"
 
 describe("AboutModal", () => {
```

Comment:
Another small issue here, I think we would have to change the name from `AboutModal` to `About` to prevent any confusion as there is no component called `AboutModal` anymore